dref/serializers.py

```diff
@@ -346,15 +357,16 @@ class DrefSerializer(NestedUpdateMixin, NestedCreateMixin, ModelSerializer):
     country_details = MiniCountrySerializer(source="country", read_only=True)
     district_details = MiniDistrictSerializer(source="district", read_only=True, many=True)
     assessment_report_details = DrefFileSerializer(source="assessment_report", read_only=True)
-    supporting_document_details = DrefFileSerializer(read_only=True, source="supporting_document")
+    supporting_document_details = DrefFileSerializer(source="supporting_document", read_only=True)
```

Comment:
Looks like we need to add additional parameters `required=False, allow_null=True`  to specify nullable fields.